client/src/views/mobile/HomeMobileView.tsx

```diff
@@ -16,6 +17,8 @@ export interface HomeMobileViewProps {
   rightAd: AdResult;
   createBrute: () => void;
   character: React.JSX.Element;
+  fixBruteAppearance: boolean;
+  setFixBruteAppearance: (params: (prev: boolean) => boolean) => void;
```

Comment:
You can use the setter type instead `React.Dispatch<React.SetStateAction<string>>`